src/Microsoft.Health.Dicom.Client/DicomWebClient.ExtendedQueryTag.cs

```diff
@@ -102,7 +104,7 @@ public async Task<DicomWebResponse<GetExtendedQueryTagEntry>> UpdateExtendedQuer
         using var request = new HttpRequestMessage(HttpMethod.Patch, uri);
         {
             request.Content = new StringContent(jsonString);
-            request.Content.Headers.ContentType = new System.Net.Http.Headers.MediaTypeHeaderValue(DicomWebConstants.ApplicationJsonMediaType);
+            request.Content.Headers.ContentType = new MediaTypeHeaderValue(DicomWebConstants.ApplicationJsonMediaType) { CharSet = Encoding.UTF8.WebName };
```

Comment:
We'll need to wait on a change from @shivakumar-ms to improve work item serialization